bitnami/external-dns/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.rbac.create }}
+{{- if and .Values.rbac.create .Values.rbac.serviceAccount.create -}}
```

Comment:
Revert this change

Reply:
Cannot revert for the same reason as https://github.com/bitnami/charts/pull/2088#discussion_r395482973

Reply:
Your suggest ?

```
{{- if and .Values.rbac.create .Values.rbac.serviceAccountName -}}
```